conpot/conpot.cfg

```diff
@@ -39,6 +39,14 @@ host = taxiitest.mitre.org
 port = 80
 inbox_path = /services/inbox/default/
 use_https = False
+
+[stix]
+port_path_list = [
+    "//conpot_template/protocols/modbus/@port",
```

Comment:
Do we really need them if they are always `//conpot_template/protocols/{protocol}/@port`?